include/AST/Decl/Decls.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef GLU_AST_DECL_DECLS_HPP
```

Comment:
put it at the root of AST
we want to include `AST/Decls.hpp` not `AST/Decl/Decls.hpp`, it's repetitive